timemachine/fe/rbfe.py

```diff
@@ -122,205 +132,146 @@ def setup_initial_states(st, host_config, temperature, lambda_schedule, seed, mi
     -------
     list of InitialStates
         Returns an initial state for each value of lambda.
-
     """
 
-    host = None
     if host_config:
-        host_system, host_masses = convert_omm_system(host_config.omm_system)
-        host_conf = minimizer.minimize_host_4d(
-            [st.mol_a, st.mol_b],
-            host_config.omm_system,
-            host_config.conf,
-            st.ff,
-            host_config.box,
-        )
-        host = (host_system, host_masses, host_conf)
+        host = setup_host(st, host_config)
+    else:
+        host = None
 
     initial_states = []
 
     # check that the lambda schedule is monotonically increasing.
     assert np.all(np.diff(lambda_schedule) > 0)
 
-    for lamb_idx, lamb in enumerate(lambda_schedule):
+    conf_a = get_romol_conf(st.mol_a)
+    conf_b = get_romol_conf(st.mol_b)
 
-        mol_a_conf = get_romol_conf(st.mol_a)
-        mol_b_conf = get_romol_conf(st.mol_b)
+    for lamb_idx, lamb in enumerate(lambda_schedule):
+        ligand_conf = st.combine_confs(conf_a, conf_b, lamb)
 
-        assert lamb >= 0.0 and lamb <= 1.0
+        # use a different seed to initialize every window,
+        # but in a way that should be symmetric for
+        # A -> B vs. B -> A edge definitions
+        init_seed = int(seed + hash(ligand_conf.tobytes())) % 10000
```

Comment:
nit: comment on reason for `% 10000`? Also, consider defining `get_init_seed_window(seed, ligand_conf) -> int`, since this is nontrivial and repeated below.

Reply:
>nit: comment on reason for % 10000? 

The constant 10000 is arbitrary, but I think I need to do something to limit this value's bit_length (reverting this commit https://github.com/proteneer/timemachine/pull/936/commits/e1f7328f01f427534d8744aab6027338e116ad09 causes a type error in one of the tests when this is passed as the `seed: int` argument to the LangevinIntegrator constructor)

Will comment inline

>Also, consider defining get_init_seed_window(seed, ligand_conf) -> int, since this is nontrivial and repeated below.

It's slightly different (hashing the conformer, vs. hashing the parameters), but yes, there's logic in common that should be extracted -- willfix

Reply:
Ah, I'd missed that the two cases are different. Feel free to disregard the suggestion to define a helper function.

Reply:
Added a comment in https://github.com/proteneer/timemachine/pull/936/commits/f3eb32dcbfeb39858180a166f8419540c1157c56 but declined to add a helper function